src/test/coins_tests.cpp

```diff
@@ -180,10 +183,10 @@ class CCoinsViewTest : public CCoinsView
         BatchWriteNullifiers(mapSproutNullifiers, mapSproutNullifiers_);
         BatchWriteNullifiers(mapSaplingNullifiers, mapSaplingNullifiers_);
 
-        mapCoins.clear();
         mapSproutAnchors.clear();
         mapSaplingAnchors.clear();
```

Comment:
This may be neither here nor there, but, along the same lines as mapCoins, I wondered if it made sense to remove the .clear() for the anchor maps.